CHANGELOG.md

```diff
@@ -26,14 +30,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - The copyright for all sources is now attributed to Open Enclave SDK contributors.
 - Update Intel DCAP library dependencies to 1.4.1.
 - Update Intel PSW dependencies to 2.6.100.2 on Windows.
-- The stack guard for OE SDK on SGX is turned on. It is also turned on for enclaves.
+- Enable /W2 to treat W2 warnings as errors for Windows builds. Additionally, fix errors which arise from enabling this flag.
+- Fix oe_random for size > 1024.
+- Add openenclave namespace to dl and crypto libraries. Fixes #2082
+- Implement host side sockets on Windows.
+- Removed code related to deprecation of strftime.
+- Stack guard for enclaves is enabled. Enclave libs and enclaves are built using `-fstack-protector-strong`.
+- Enclave libs and enclaves are built using `-gc-sections`.
+- Overhaul OCaml build system to use esy and upgraded OCaml version with reproducibility.
+- Bump ansible from 2.8.0 to 2.8.2 in /scripts/ansible
```

Comment:
Nit: I would replace "Bump" for "Update" for consistent word choice, and also pay attention to capitalization of product names:

```
- Update Ansible dependency from 2.8.0 to 2.8.2 in /scripts/ansible.
```